pkg/wasm/convert.go

```diff
@@ -38,23 +41,104 @@ var (
 	allowHTTPTypedConfig = &anypb.Any{
 		TypeUrl: "type.googleapis.com/envoy.extensions.filters.http.rbac.v3.RBAC",
 	}
-	allowNetworkTypedConfig = &anypb.Any{
-		TypeUrl: "type.googleapis.com/envoy.extensions.filters.network.rbac.v3.RBAC",
-	}
+	denyHTTPTypedConfig, _ = anypb.New(&httprbac.RBAC{
+		Rules: &rbacv3.RBAC{
+			Action: rbacv3.RBAC_DENY,
+			Policies: map[string]*rbacv3.Policy{
+				"wasm-default-rule": {
+					Permissions: []*rbacv3.Permission{
+						{
+							Rule: &rbacv3.Permission_Any{
+								Any: true,
+							},
+						},
+					},
+					Principals: []*rbacv3.Principal{
+						{
+							Identifier: &rbacv3.Principal_Any{
+								Any: true,
+							},
+						},
+					},
+				},
+			},
+		},
+	})
+
+	allowNetworkTypeConfig, _ = anypb.New(&networkrbac.RBAC{
+		Rules: &rbacv3.RBAC{
+			Action: rbacv3.RBAC_ALLOW,
+			Policies: map[string]*rbacv3.Policy{
```

Comment:
Why is this changed?

Reply:
Before the modification, this was an empty configuration. Envoy requires that stat_prefix cannot be empty, so it rejects this configuration. However, the httpfilter does not have this requirement.

After testing, I found that only this type of configuration can achieve the purpose of allow-all. Therefore, I also made changes here.

Reply:
resolved